web_frontend/config/settings/local.py

```diff
@@ -9,7 +9,18 @@
 MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
 INSTALLED_APPS += ('debug_toolbar', )
 
-INTERNAL_IPS = env.tuple('DJANGO_INTERNAL_IPS', default=('127.0.0.1',))
+
+from fnmatch import fnmatch
+
+
+class glob_list(list):  # noqa
+    def __contains__(self, key):
+        for elt in self:
+            if fnmatch(key, elt):
+                return True
+        return False
+
+INTERNAL_IPS = glob_list(env.tuple('DJANGO_INTERNAL_IPS', default=('127.0.0.1',)))
```

Comment:
`itertools.chain` requires `self.__next__()` to work.

My approach seems to work with `collections.ChainMap` instead of `itertools.chain`, though. :-)